components/shared/modals/ModalBody.vue

```diff
@@ -68,33 +74,37 @@ const props = withDefaults(
     contentClass?: string
     scrollable?: boolean
     customSkeletonTitle?: string
+    estimatedTime?: number
   }>(),
   {
     modalWidth: '25rem',
     modalMaxWidth: '30rem',
     modalMaxHeight: '70vh',
     scrollable: true,
     contentClass: 'pt-4 pb-5 px-6',
+    estimatedTime: undefined,
   },
 )
 
 const { $i18n } = useNuxtApp()
+const { estimatedTime: formattedEstimatedTime, start: startEstimatedTime } =
+  useEstimatedTime({ duration: computed(() => Number(props.estimatedTime)) })
```

Comment:
![image](https://github.com/kodadot/nft-gallery/assets/22791238/5371adc2-5a3c-4c82-8f85-a51dfa745935)

Reply:
yup, knew that, I was trying to not get a linting error since it could be undefined, changed it , pls check